wasmedge-ggml-llama-interactive/src/main.rs

```diff
@@ -28,10 +28,26 @@ fn main() {
     let system_prompt = String::from("<<SYS>>You are a helpful, respectful and honest assistant. Always answer as short as possible, while being safe. <</SYS>>");
     let mut saved_prompt = String::new();
 
+    // Set options to input with index 1
+    let options = r#"{"enable-log": true, "ctx-size": 1024}"#;
```

Comment:
I think we should have a document for this. There is no way to know how to use these `options`.

Reply:
Sure, will update document for the options.

Reply:
Added to README.md